### PR TITLE
Further Comments standardization #3901

### LINKED ISSUE #3901: Further Comments standardization
Continuation of #3838, now handling the finer details.

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #3901 
HTML to be fixed after all changes are agreeable.

### CODE REVIEW

**Comment by junhaoyap** [comment]
I'll take a look soon, tonight / tomorrow.

**Comment by wkurniawan07** [comment]
I need to re-do this on the updated master (same PR) so can delay the review until tomorrow

**Comment by wkurniawan07** [comment]
Ready for review. When God mode can be used, regenerating the HTML files is really a child play :+1:

**Comment by wkurniawan07** [comment]
@junhaoyap please review if you have time xP gonna need this if I want to migrate Records

**Comment by junhaoyap** [comment]
Ok looking at it now, suddenly become 2000 lines o.o

File: src/main/webapp/js/omniComment.js [hunk 1]
```diff
@@ -410,7 +410,7 @@ function deleteComment(commentIdx){
     }
 }
 
-function enableEdit(commentIdx){
+function enableEdit(commentIdx, unusedParameter) {
```
Comment (junhaoyap) [inline src/main/webapp/js/omniComment.js]: why is there this unusedParameter? xD

File: src/main/webapp/jsp/instructorSearch.jsp [hunk 2]
```diff
@@ -35,4 +35,4 @@
         </search:studentsSearchResults>
     </c:if>
     
-</ti:instructorPage>
+</ti:instructorPage>
```
Comment (junhaoyap) [inline src/main/webapp/jsp/instructorSearch.jsp]: Odd, this looks the same to me, how come GitHub shows a diff?

File: src/main/webapp/WEB-INF/tags/instructor/comments/visibilityOptions.tag [hunk 3]
```diff
@@ -20,7 +29,7 @@
     <div id="visibility-options${commentIdx}" class="panel panel-default"
         style="display: none;">
         <div class="panel-heading">Visibility Options</div>
-        <table class="table text-center text-color-black">
+        <table class="table text-center" style="color: #000;">
```
Comment (junhaoyap) [inline src/main/webapp/WEB-INF/tags/instructor/comments/visibilityOptions.tag]: Why not use text-color-black, it's more descriptive than the RGB value of 00 00 00, plus using CSS > using inline styles, or is it more HTML changes needed when we use classes?

File: src/main/webapp/WEB-INF/tags/instructor/comments/commentsForStudentsPanel.tag [hunk 4]
```diff
@@ -41,39 +47,27 @@
                                             onclick="return deleteComment('${commentIdx}');"
                                             data-toggle="tooltip"
                                             data-placement="top"
-                                            title=""
-                                            data-original-title="<%= Const.Tooltips.COMMENT_DELETE %>"
+                                            title="<%= Const.Tooltips.COMMENT_DELETE %>"
```
Comment (junhaoyap) [inline src/main/webapp/WEB-INF/tags/instructor/comments/commentsForStudentsPanel.tag]: Good catch, we need to fix this for other files as well such as

screen shot 2015-07-09 at 5 35 32 pm

whoever copy pasted stuff just tried to do what bootstrap would have done manually

File: src/main/webapp/js/omniComment.js (hunk 1 above)
Comment (wkurniawan07) [inline src/main/webapp/js/omniComment.js]: I wish I don't have to add it, but the other `enableEdit` function (in `instructorStudentRecords.js`) has 2 params XP

File: src/main/webapp/WEB-INF/tags/instructor/comments/visibilityOptions.tag (hunk 3 above)
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/instructor/comments/visibilityOptions.tag]: Yeah, this is the one that's different compared to others. FRC used `style="color: #000"` as well. We can settle this is another issue I guess?

**Comment by wkurniawan07** [comment]
You can believe me when I said almost all of the lines changed are caused by the last commit :stuck_out_tongue:

File: src/main/webapp/WEB-INF/tags/instructor/comments/visibilityOptions.tag (hunk 3 above)
Comment (junhaoyap) [inline src/main/webapp/WEB-INF/tags/instructor/comments/visibilityOptions.tag]: Yupp, let's open another issue for this, text-color-black is definitely a better choice than RGB 000.

**Comment by junhaoyap** [comment]
Yeah I looked through the changes, it's good it added back some stuff that were omitted, removed that weird do what bootstrap is gonna do anyway thing etc, we can merge this but remember to open a few issues before we forget:

1. Don't use RGB 000 but use text-color-black
2. Let's look into the enableEdit and the unused params, I'm sure there's a way we can fix that weird thing
3. Revert all the data-original-title="some string" and title="" from our tag files, they're supposed to be done by bootstrap, we're not supposed to say title="" and jump ahead of bootstrap

**Comment by wkurniawan07** [comment]
Feel free to open 1 and 3, but hold them until I finish making a shared Comment tag because after that all you need is changing that one tag (and a bunch of HTML files XP but with God mode this is a child's play)

**Comment by junhaoyap** [comment]
Just to be sure, the name= without the "" quotes are automatically fixed by the browsers so no change in the html files right?

**Comment by wkurniawan07** [comment]
Yeah that one's done by browser, so no change in HTML files for that aspect.

**Pull request merged**